UnityProject/Assets/Scripts/Weapons/WeaponNetworkActions.cs

```diff
@@ -73,7 +73,8 @@ public void CmdUnloadWeapon(GameObject weapon)
 			playerMove.isGhost ||
 			!victim ||
 			!playerScript.playerNetworkActions.SlotNotEmpty(slot) ||
-			!playerScript.playerHealth.serverPlayerConscious
+			playerScript.playerHealth.IsCrit ||
+			playerScript.playerHealth.IsDead
```

Comment:
You're better off adding these two to PlayerScript.canNotInteract() method and using it here

Reply:
fixed